### ISSUE TITLE
Change icon of default buttons

### ISSUE THREAD

**Issue by SanderM2** [berrywhite96/lovelace-shutter-row#30]
Is it possible to change the icon of the default buttons?

I once requested this feature but my request has been closed with the message that it has been implemented in version 0.3.0.
Except that I can't find any documentation / example for this?

I do see that custom buttons were introduced but I just want to change the up/stop/down buttons to mdi:arrow-up-bold-circle-outline, mdi:stop-circle-outline and mdi:arrow-down-bold-circle-outline respectively. (in order to match other UI elements in my dashboard)

**Comment by berrywhite96** [comment]
I think there was a misunderstanding: state color is a feature of this card which changes the icon color respectively to the state.

This card won't get any new features, as I am working more or less atm on a new card. Just trying to keep it as long a possible life.

Do you know `card-mod`? Should be easy to change the color with it.

**Comment by SanderM2** [comment]
I do not want to change the color but maybe you meant to say that I can change the icon with card-mod?

**Comment by berrywhite96** [comment]
> Sad to read that this card won't get any new features. Are you creating a new covers card then? And is it already available for testing/beta users?

It will support nearly every entity type and I will try to fit in all known features from this card too.
No, I am not convinced atm with the main handling, I am breaking my head about this for months already.

> I do not want to change the color but maybe you meant to say that I can change the icon with card-mod?

My bad, this happens when you try to answer in between gym sets 🫣.
Still don't know which feature you saw, you can't change icons. You can just customize all actions of all three main buttons.

I need to look at home, but I am quite sure the icons are hardcoded.

**Comment by SanderM2** [comment]
Ok. So no more feature updates to this card for now so I decided to not update anymore and change it to my likings for now.
So far I changed the icons to my likings by changing the mdiChevronDown, mdiChevronUp and mdiStop variables inside the script.
So far, so good.

The only thing that's bothering me now is the fact that the slide no longer has working tooltip and won't show the position before letting go your finger/mouse when changing the position.

This is not a new feature but a bugfix request. I already opened another issue for that. I hope you can fix this or at least point me in the right direction on how to get it fixed?

**Comment by berrywhite96** [comment]
Yes this is correct, just import the icons from the library and use it at the point where it renders the buttons.

I saw that issue and I never notice it myself until today. But I noticed one thing mini media player card, which has the same slider as this card: it's very buggy with the slider. Seems like they reimplemented the tool tip, but not that perfomant.

I need to look if there is a quick fix possible, but I don't have high hopes.
